### PR TITLE
add image and number of books to categories display

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by zenith19** [comment]
ok

File: app/uploaders/image_uploader.rb [hunk 1]
```diff
@@ -33,9 +33,9 @@ def store_dir
   # end
 
   # Create different versions of your uploaded files:
-  # version :thumb do
-  #   process :resize_to_fit => [50, 50]
-  # end
+  version :thumb do
+    process :resize_to_fit => [80, 80]
```
Comment (dieunb) [inline app/uploaders/image_uploader.rb]: Convention

Comment (MuhammadTamzid) [inline app/uploaders/image_uploader.rb]: Sorry :D

**Pull request merged**